libsplinter/src/biome/rest_api/actix/key_management.rs

```diff
@@ -278,3 +278,192 @@ fn handle_patch(
         }))
     })
 }
+
+/// Defines a REST endpoint for managing keys including fetching and deleting a user's key
+pub fn make_key_management_route_with_public_key(
+    rest_config: Arc<BiomeRestConfig>,
```

Comment:
Can we get rid of these Arc?

Reply:
I added a story to review the use of Arc within Biome: https://zenhub.cglcloud.com/app/workspaces/product-core-5d37298b0add7306b30efde2/issues/digitalsupplychain/product-core/2056, as I don't think this is necessary within the scope of this story/pr